src/scss/_mixins.scss

```diff
@@ -115,7 +92,7 @@
 
 		&:before {
 			@include oIconsBaseStyles();
-			background-image: url('https://www.ft.com/__origami/service/image/v2/images/raw/ftlogo-v1:#{$logo}?source=o-footer-services');
+			background-image: url('https://www.ft.com/__origami/service/image/v2/images/raw/#{$logo}?source=o-footer-services');
```

Comment:
Is it possible to URL-encode the logo? Or do we think that's not a good idea?

Reply:
Good shout. There's not a built in function so it's [a bit of a pain](https://github.com/Financial-Times/o-brand/blob/master/scss/_private-functions.scss#L144).

Reply:
If it's annoying then maybe not? We can always document that it has to be pre-encoded

Reply:
Looking for a library to do this... we could create a little one if it doesn't exist

Reply:
I couldn't find an ideal library, so for now I've done it manually within this component